File: app/src/main/java/com/durbar/bangabandhuplay/data/model/ott_content/Pivot.java
package com.durbar.bangabandhuplay.data.model.ott_content;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Pivot {
    @SerializedName("ott_content_id")
    @Expose
    public Integer ottContentId;
    @SerializedName("cast_and_crew_id")
    @Expose
    public Integer castAndCrewId;
    @SerializedName("role")
    @Expose
    public String role;

    public Pivot(Integer ottContentId, Integer castAndCrewId, String role) {
        this.ottContentId = ottContentId;
        this.castAndCrewId = castAndCrewId;
        this.role = role;
    }

    public Integer getOttContentId() {
        return ottContentId;
    }

    public Integer getCastAndCrewId() {
        return castAndCrewId;
    }

    public String getRole() {
        return role;
    }
}

File: app/src/main/java/live/durbar/bangabandhuapp/data/model/ott_content/Pivot.kt
package live.durbar.bangabandhuapp.data.model.ott_content

import com.google.gson.annotations.SerializedName

data class Pivot(
    @field:SerializedName("ott_content_id")
    val ottContentId: Int? = 0,
    @field:SerializedName("cast_and_crew_id")
    val castAndCrewId: Int? = 0,
    @field:SerializedName("role")
    val role: String? = null
)
